services/government-frontend/docker-compose.yml

```diff
@@ -44,3 +44,17 @@ services:
       VIRTUAL_HOST: draft-government-frontend.dev.gov.uk
       PLEK_HOSTNAME_PREFIX: "draft-"
       HOST: 0.0.0.0
+
+  government-frontend-dummy:
+    <<: *government-frontend
+    depends_on:
+      - nginx-proxy
+    environment:
+      GOVUK_WEBSITE_ROOT: https://www.gov.uk
+      PLEK_SERVICE_CONTENT_STORE_URI: https://govuk-content-store-examples.herokuapp.com/api
+      PLEK_SERVICE_STATIC_URI: assets.publishing.service.gov.uk
+      VIRTUAL_HOST: dummy-government-frontend.dev.gov.uk
```

Comment:
Should we keep this consistent with the stack name, ie. `government-frontend-dummy.dev.gov.uk`?

Reply:
So the naming here is consistent with e.g. 'draft-government-frontend'. We could also just called it 'government-frontend', as long as people aren't wanting to spinup this stack and the live stack at the same time. Would that be better?